content/blog/2021-02-10-onboarding-resource-roundup/contents.lr

```diff
@@ -0,0 +1,67 @@
+title: InvenioRDM Onboarding Resource Roundup
+---
+author: Sara Gonzales
+---
+body:
+
+Congratulations! Your team has just joined the InvenioRDM open source project and is eager to contribute as much as possible. With a mixture of dedicated developers, resource curators, subject matter experts, and various leadership stakeholders, your team simply needs to know about the available resources and how to get started. Read on!
+
+
+### Developers
+
+Whether you would like to install and customize InvenioRDM, contribute on the main project sprints, or take on development of a module locally to contribute back to the project, there are a few important points of entry to participation:
+
+- [Roadmap](https://inveniosoftware.org/products/rdm/roadmap/): Familiarize yourself with the overall project timeline and goals. Stable releases will be made through Summer 2021 on a monthly/bi-monthly basis
+
+- [Development Roadmap](https://github.com/inveniosoftware/product-rdm/milestones?direction=asc&sort=due_date&state=open): Get to know the specific milestones for 2021
+
+- [Sprintboard](https://github.com/orgs/inveniosoftware/projects/63): See the planning board for the current sprint
+
+- [Documentation](https://inveniordm.docs.cern.ch/): Read the official project documentation
+
+- [Discord](https://discord.gg/8qatqBC): Ready to get started? Ask Lars for an invitation to the project Discord (chat/video platform) and introduce yourself in either the general or newcomers-help channel. A daily developer standup takes place on Discord at 15:15 UTC
+
+- [Discourse](https://invenio-talk.web.cern.ch/): Want to provide feedback, but in a less time-sensitive manner than a chat? Either join or start a thread in the Discourse discussion forum
+
+- [Development Setup](https://inveniordm.docs.cern.ch/develop/): Install a development version
+
+- [Development Workflows](https://inveniordm.docs.cern.ch/maintenance/development_workflow/): Learn about the development workflows used
+
+### Community Members
+
+All stakeholders at your institution are welcome to contribute to InvenioRDM, whether their contribution is code-based or not. To create a tool to best serve the community, outlining stakeholder requirements will be vital:
+
+ - [Demo site](https://inveniordm.web.cern.ch/): Access the latest sandbox version of InvenioRDM here. Create an account to test record creation
+
+ - [Discourse](https://invenio-talk.web.cern.ch/): Provide feedback on testing here. Use the [Feedback](https://invenio-talk.web.cern.ch/c/site-feedback/2) category or start a new thread wherever you like
```

Comment:
Hmm... I am only going to check the monthly feedback thread referred above for time (quicker), specificity (know which release) and singularity (if same issue encountered multiple times, others can see it in the same thread). I also think the feedback section is for the framework as a whole (i.e. not just InvenioRDM).

Could be fine if others are checking these threads, but it's setting up additional work ;)

Reply:
Good to know. Are others checking? I could also check once/month and alert you to any posts.

Reply:
I don't know if others are checking it regularly. Discourse sends a weekly digest email if you haven't visited the site in the last week to keep you appraised of posts I believe. The responsiveness is therefore not as quick, but the feedback is more permanent and focused than on Discord (I know Discord and Discourse have similar names...).GitHub tickets are more technical. In the end we have feedback areas for everyone, but it's hard for developers to keep track of them. That's why I try to limit myself...